inlong-manager/manager-dao/src/main/resources/mappers/StreamSourceEntityMapper.xml

```diff
@@ -120,6 +120,19 @@
             </if>
         </where>
     </select>
+    <select id="selectDataAddTaskCount" resultType="java.lang.Integer">
+        select count(1)
+        from stream_source
+        <where>
+            <if test="groupId != null and groupId != ''">
+                and inlong_group_id = #{groupId, jdbcType=VARCHAR}
```

Comment:
Is there any case that we need to count tasks only by stream Id?

Reply:
It may not be possible to search based on groupid and streamid.